contrib/containerized-pbench/README.md

```diff
@@ -1,81 +1,118 @@
-# containerized-pbench
-
-## Openshift
-
-We can either build an image using the following commands or pull the image from dockerhub
-
-Build the image from source like
+## Containerized pbench
+This directory contains OPenShift templates, dockerfiles needed to build and run containerized pbench.
 
+### Prepare the nodes you want to monitor
+The nodes need to have pbench-agent, collectd images on it. The qcow images generated by the image provisioner already has the images baked in. In case the nodes doesn't have the images, please pull them from dockerhub:
+#### pbench-agent
 ```
-$ cd containerized-pbench/dockerfiles-pod
-$ docker build -t pbench-agent .
+$ docker pull ravielluri/image:agent
+$ docker tag ravielluri/image:agent pbench-agent:latest
 ```
 
-Pull the image from dockerhub, if the image is not present on the openshift nodes, it is pulled automatically from dockerhub or registry
+### Label the nodes with a type=pbench label
+```   
+$ oc label node <node> type=pbench
+```
 
+### Create a service account and add it to the privileged scc
 ```
-$ docker pull chaitanyaenr/pbench-agent
+$ oc create serviceaccount useroot
+$ oc adm policy add-scc-to-user privileged -z useroot
 ```
 
-Run the following commands on openshift master
-
-To launch a pbench-pod on all the nodes:
+### Create pbench-agent pods and patch it
 ```
-$ oc create -f containerized-pbench/openshift-templates/pbench-agent-daemonset.yml
+$ oc create -f openshift_templates/performance_monitoring/pbench/pbench-agent-daemonset.yml
+$ oc patch daemonset pbench-agent --patch \ '{"spec":{"template":{"spec":{"serviceAccountName": "useroot"}}}}'
 ```
 
-By default pod is not scheduled on the master node since the scheduling is disabled. Since we are using daemonset, pods created by the Daemon controller have the machine already selected (.spec.nodeName is specified when the pod is created, so it is ignored by the scheduler) which means that the unschedulable field of a node is not respected by the DaemonSet controller and DaemonSet controller can make pods even when the scheduler has not been started. For this to work we have to create a service account and add it to the privileged scc like
+## Prepare the jump host to run pbench-controller
```

Comment:
Is something missing here? How do you prep the jump host to run pbench-controller?